core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -47,6 +47,8 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         // Raw collateral value. This value should never be accessed directly -- always use _getCollateral().
         // To add or remove collateral, use _addCollateral() and _removeCollateral().
         FixedPoint.Unsigned rawCollateral;
+        // Tracks pending transfer requests. A transfer request is pending if `transferRequestPastTimestamp != 0`.
+        uint256 transferRequestPassTimestamp;
```

Comment:
I unintentionally forked this branch off of the branch I used in #1314 so once that PR is merged this file will show a much smaller diff

Reply:
done